src/large.c

```diff
@@ -368,8 +368,10 @@ large_salloc(tsdn_t *tsdn, const extent_t *extent) {
 }
 
 void
-large_prof_info_get(const extent_t *extent, prof_info_t *prof_info) {
-	extent_prof_info_get(extent, prof_info);
+large_alloc_prof_info_get(const extent_t *extent, prof_info_t *prof_info) {
```

Comment:
nit: Move to large.h and make it inline?

Reply:
There's unfortunately no `large.h`. We only have `large_externs.h` and all `large_*` functions are not inlined.